.prettierrc

```diff
@@ -2,5 +2,6 @@
   "arrowParens": "always",
   "printWidth": 130,
   "singleQuote": true,
-  "trailingComma": "all"
+  "trailingComma": "all",
+  "endOfLine": "auto"
```

Comment:
Why set this to `auto`? The recommendation/default is `lf`?
https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-endofline-to-lf-7435httpsgithubcomprettierprettierpull7435-by-kachkaevhttpsgithubcomkachkaev

Reply:
This was breaking the build on wins. I suppose it is possible we can fix it on a wins machine but I did not have access to one.